### PR TITLE
Add 'persist' parameter to execute events added after the event has b…

### PR DESCRIPTION
**Pull request by senica**
…een triggered

Replacement for https://github.com/riot/observable/pull/20

This does not use namespaces.

The purpose of this is to capture events that should be triggered in highly dynamic environments and execute them. This prevents race conditions.

This is helpful in many situations, but a couple I've used it in are:
- my page is setup and waiting for websocket data from the server. Once a connection is made, trigger a connect event. There are times, when the websocket will connect before the page finishes loading. I still want the connect events to execute and ALWAYS execute everytime the connect event happens.
- mounting tags asynchronously from the server and one tag triggers an event that another tag needs to execute. With the persist, I don't have to worry about what order my tags are loaded in.

Basically, this solves any issue where race conditions are an issue with your events.

Example:

```
var app = {};
riot.observable(app);
app.trigger('connect');
app.on('connect', function(){
  // this will still execute even though it is setup AFTER the trigger
}, true); // <- note the persist boolean.
```

### CODE REVIEW

**Comment by senica** [comment]
Whoops, wrong branch.
